subsys/shell/Kconfig

```diff
@@ -297,13 +297,52 @@ config SHELL_LOG_BACKEND_CUSTOM
 	  It allows to provide custom implementation of multiplexing logging messages
 	  with shell data.
 
+config SHELL_LOG_OUTPUT_TIMESTAMP
+	bool "Log output timestamp"
+	default y
+	depends on SHELL_LOG_BACKEND
+	help
+	  Enable timestamp in log output.
+
 config SHELL_LOG_FORMAT_TIMESTAMP
 	bool "Format timestamp"
 	default y
 	depends on SHELL_LOG_BACKEND
 	help
 	  Enable timestamp formatting.
 
+config SHELL_LOG_OUTPUT_LEVEL
+	bool "Log output level"
+	default y
+	depends on SHELL_LOG_BACKEND
+	help
+	  Enable log level in log output.
+
+config SHELL_LOG_OUTPUT_CRLF_NONE
+	bool "Log output CRLF none"
+	depends on SHELL_LOG_BACKEND
+	help
+	  Do not modify CRLF characters in log output.
+
+config SHELL_LOG_OUTPUT_CRLF_LFONLY
+	bool "Log output CRLF LF only"
+	depends on SHELL_LOG_BACKEND
+	help
+	  Convert CRLF to LF in log output.
+
+config SHELL_LOG_OUTPUT_THREAD
+	bool "Log output thread"
+	default y
+	depends on SHELL_LOG_BACKEND
```

Comment:
Put a if SHELL_LOG_BACKEND around these all, so we dont have to do the depends on on every entry